pyproject.toml

```diff
@@ -9,6 +9,14 @@ requires = [
 ]
 build-backend = 'setuptools.build_meta'
 
+[tool.cibuildwheel]
+
+[tool.cibuildwheel.linux]
+before-all = "yum -y install fftw-devel; git clean -xfd -e wheelhouse"
+
+[tool.cibuildwheel.macos]
+before-all = "brew install fftw; git clean -xfd -e wheelhouse"
```

Comment:
```suggestion
before-all = "yum -y install fftw-devel; git clean -xfd -e dist"

[tool.cibuildwheel.macos]
before-all = "brew install fftw; git clean -xfd -e dist"
```
Not sure if this is the fix, but the publish workflow sets the output directory to `dist` instead of the cibuildwheel default of `wheelhouse` https://github.com/OpenAstronomy/github-actions-workflows/blob/1dbaaee6ed319f8ab85dbad6389c5dcf2b50b503/.github/workflows/publish.yml#L144

Reply:
Actually, I think that should fix it.

Reply:
Thanks, I will try that. Surprised that its ok for linux.